Export.h.in

```diff
@@ -0,0 +1,138 @@
+/** @file
+    @brief Automatically-generated export header - do not edit!
+
+    @date 2016
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2016 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef @EXPORT_BASENAME@_EXPORT_H
```

Comment:
`@EXPORT_BASENAME@` gets substituted by the value of `${EXPORT_BASENAME}` in CMake at the time of the `configure_file` command. Here, we're looking for something like `OSVR_RENDERMANAGER` - whatever the prefix of `_EXPORT` is. It should be set before the `configure_file` command in CMake.

This wasn't part of the change in Core because we were already using variables named that to send parameters to GenerateExportHeader.